README.md

```diff
@@ -57,7 +57,23 @@ self.addEventListener('periodicsync', event => {
 
 # Security and Privacy
 
-TODO
+One-shot Background Sync allows the web page’s logic to live a little longer (service worker execution time) after the page has been closed by the user. Periodic Background Sync extends it to potentially infinite time thereafter. Here are some privacy concerns:
+
+* There is no notification to the user.
+* A periodic sync can be enabled while the user is connected to one network, and the sync event
+can be fired later when they're connected to another network. This can cause inadvertent leakage 
+of browsing history on an unintended network. This concern applies to other background tasks such as one-shot Sync and Background Fetch as well.
+* Location tracking: The user’s IP address can be revealed to the website every time the periodic task is run. This risk is also present with any tasks run in a service worker, but Periodic Sync allows persistent tracking. Note that this risk is also present with Push messages,
+but they require explicit user permission.
+
+The following steps should be taken by the user agent to mitigate these privacy concerns:
+* Limit the number of times the periodic tasks are run to a few times a day.
```

Comment:
Why? If there's an open tab on a desktop environment it'd be entirely reasonable to run at a much higher cadence.

Reply:
Updated wording to reflect this. Note that all of this is advisory, indicated by the "The following steps *should* be taken by the user agent" line. I can remove all of these recommendations if it doesn't make sense to note them here. LMK what you think.